File: Programmers/Lv.1/행렬의덧셈.py
def solution(arr1, arr2):
    answer = []
    
    for i in range(len(arr1)):
        arr_sum = []
        for j in range(len(arr1[i])):
            arr_sum.append(arr1[i][j] + arr2[i][j])
        answer.append(arr_sum)
    
    return answer

# def solution(A,B):
#     answer = [[c + d for c, d in zip(a, b)] for a, b in zip(A,B)]
#     return answer

File: Programmers/Lv.1.Kotlin/행렬의덧셈.kt
class Solution {
    fun solution(arr1: Array<IntArray>, arr2: Array<IntArray>): Array<IntArray> {
        var answer = arrayOf<IntArray>()

        // arr1의 인덱스와 해당 인덱스 배열
        answer = arr1.mapIndexed { indexArr, nums ->
            // arr1의 각 배열의 인덱스와 값
            nums.mapIndexed {indexNums, i ->
                i + arr2[indexArr][indexNums]
            }.toIntArray()
        }.toTypedArray()

        return answer

        // return Array(arr1.size) { row ->
        //     IntArray(arr1[0].size) { col ->
        //         arr1[row][col] + arr2[row][col]
        //     }
        // }
    }
}